content/correspondence/getting-started/developer-guides/systemprovider/_index.nb.md

```
@@ -17,7 +17,7 @@ For å komme i gang som systemleverandør følg denne veiledningen:
 For å kunne autentisere og sikre at du kan utføre operasjoner via meldings-APIet, må Altinn gi deg tilgang på de scopes du trenger. Dette sikrer at kun autoriserte klienter kan sende og motta filer, og opprettholder dermed sikkerheten i tjenesten. Følgende scopes brukes for å motta meldinger:
 - `altinn:correspondence.read`
 
-For å få Altinn API-nøkkel og scopes må du sende en forespørsel til: servicedesk@altinn.no 
+For å få tilgang til scopes scopes må du sende en forespørsel til: servicedesk@altinn.no 
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)